src/test/common/steps/complete-flow.js

```diff
@@ -49,3 +53,11 @@ async function submitApplicationWithStatus (status) {
   await setupWiremockMappingsWithStatus(status)
   await pages.genericPage.submitForm()
 }
+
+async function submitApplicationForUpdatedClaim () {
```

Comment:
This function is initially confusing until you realise that the flow in the application is identical for the update process as to the new applicant process, all we're doing is simulating a different response from Wiremock to mimic that the claim was updated. It feels as though we should do better, but we simply can't. Oh well.